device/mellanox/x86_64-nvidia_sn4280_simx-r0/ACS-SN4280/buffers_defaults_t1.j2

```diff
@@ -0,0 +1,39 @@
+{#
+    Copyright (c) 2020-2024 NVIDIA CORPORATION & AFFILIATES.
+    Apache-2.0
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+#}
+{% set default_cable = '5m' %}
+{% set ingress_lossless_pool_size =  '20627456' %}
```

Comment:
Can you share the information based on which the buffer numbers are calculated?

Reply:
The buffer/qos/pg configuration is currently copied from 4700. This is temporary.
Once we have these configs defined for 4280, I'll make the x86_64-nvidia_sn4280_simx-r0/ACS-SN4280 a link to the x86_64-nvidia_sn4280-ro/ACS-SN4280